setup.py

```diff
@@ -1,11 +1,14 @@
 from setuptools import setup
 
+# Version comes directly from source w/o importing
+exec(open('closeio_api/_version.py').read())
```

Comment:
Is there some reason we can't do something like `from _version import __version__`?